Libraries/Hotcakes.PaypalWebServices/RestPaypalApi.cs

```diff
@@ -0,0 +1,288 @@
+#region License
+
+// Distributed under the MIT License
+// ============================================================
+// Copyright (c) 2019 Hotcakes Commerce, LLC
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software 
+// and associated documentation files (the "Software"), to deal in the Software without restriction, 
+// including without limitation the rights to use, copy, modify, merge, publish, distribute, 
+// sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is 
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in all copies or 
+// substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR 
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE 
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, 
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN 
+// THE SOFTWARE.
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using com.paypal.soap.api;
+using PayPalCheckoutSdk.Core;
+using PayPalCheckoutSdk.Orders;
+using PayPalCheckoutSdk.Payments;
+using PayPalHttp;
+using Capture = PayPalCheckoutSdk.Orders.Capture;
+using LinkDescription = PayPalCheckoutSdk.Orders.LinkDescription;
+using Money = PayPalCheckoutSdk.Orders.Money;
+
+namespace Hotcakes.PaypalWebServices
+{
+    /// <summary>
+    ///     Summary description for PayPalAPI.
+    /// </summary>
+    [Serializable]
+    public class RestPayPalApi
+    {
+        private string clientId; 
+        private string secret; 
+        private string mode;
+
+        public RestPayPalApi(string clientId, string secret, string mode)
+        {
+            this.clientId = clientId;
+            this.secret = secret;
+            this.mode = mode;
+        }
+
+        public HttpClient client()
+        {
+            PayPalEnvironment environment;
+
+            if (mode == "Sandbox")
```

Comment:
Please make static strings like this a `constant`.

Reply:
OK done it.